first_timers/run.py

```diff
@@ -4,50 +4,63 @@
 import os
 import sys
 import json
+import requests
+import warnings
 import first_timers as FT
 
+
 def updateDB(all_issues, db_path):
     """Truncate and write the new list of issues in the DB."""
-    with open(db_path, 'w') as dbFile:
+    with open(db_path, "w") as dbFile:
         json.dump(FT.limit_issues(all_issues), dbFile, indent=2)
 
+
 @click.command()
-@click.option('--only-save',
-        is_flag=True,
-        help='Do not post any tweets, just populate the DB.')
-@click.option('--db-path',
-        prompt='Database file',
-        default='data/db.json',
-        help='Old issues in a JSON list.')
-@click.option('--create',
-        is_flag=True,
-        help='Pass if the DB file should be created.')
-@click.option('--creds-path',
-        prompt='Credentials file',
-        default='',
-        help='File which contains Twitter account credentials. '
-             'Not needed if only saving to DB.')
-@click.option('--debug',
-        is_flag=True,
-        help='Run in debug mode (does not tweet).')
+@click.option(
```

Comment:
One small change, I see you have updated all single quotes `'` to double quotes `"` in this file.

Can we change it back to single quotes `'` as other files uses single quotes and we want to keep it consistent.

Reply:
So sorry! Linted code accidentally.